### PR TITLE
Project Movies by Maria & Tina, Team Foxes.

### PR DESCRIPTION
**Pull request by TinaJuren**
Link to our deployed project: 
https://friendly-gates-4a725d.netlify.app/

Reflections on how this week's project turned out: 
We worked together using live share. If we had more time we would like to finish the styling. 

Things we like to have clarified or explained in more detail:  
How to use the when URL stored as a constant stored in urls.js, now we've inserted the link to the API directly in the fetch. See row 6 and 13 in MovieDetails.js.

### CODE REVIEW

**Review by Irina-web-dev — APPROVED** [review]
Hi Maria and Tina! It was a pleasure to review your code 😇 It is a very well built site and you should be proud! ✨

Just a heads up, your background picture (hero image) in the movie details file doesn´t cover all the container on the desktop view and we think that it might be because you have a div for the image. Maybe you could remove the div and put the position of the article-container relative and the position of the hero image absolute. 

And we also saw that the overview text mounts on the small picture when you go over 1400px width (we are still on the movie details page). Maybe you can try adding some margin-left. 

We hope it helps 🤞

Good job! 💪🏼

File: code/src/pages/MovieDetails.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { Link } from 'react-router-dom'
+
+
+//import { URL_DETAILS } from 'reusable/urls'
```
Comment (KarolCislowski) [inline code/src/pages/MovieDetails.js:6]: you just need to export data from another file to be able to import it here.

**Review by KarolCislowski — APPROVED** [review]
Nice project but you need to make some minor adjustments in the `details` layout.
image

**Comment by KarolCislowski** [comment]
Hello, it's your project. Do you remember me? I may be imperfect, but I believe you can still change that before you sit down to your final masterpiece!
image

**Pull request closed without merging**